src/styles.js

```diff
@@ -27,4 +27,13 @@ export const styles = StyleSheet.create({
   imageContainer: {
     alignItems: 'center',
   },
+  container: {
+    backgroundColor: 'crimson',
```

Comment:
The same about colors and constants. As an example from our project:
Take a look at the file `src/theme/color/primary.color.js`, then it is used in `src/theme/competency/competency.color.js`